### ISSUE TITLE
Living persons at a certain date

### ISSUE THREAD

**Issue by pcpatfire** [ShammyLevva/FTAnalyzer#179]
Hi, I am not sure if there is a way to do it with your app. I would like to list all the living persons at a certain date, for instance on 1 March 1750, or within a date range, for instance, between 1 March 1740 and 1 March 1750. In my case, it would be useful to create the "stautus" of the pupulation of a place at a certain date (a kind of "census").
Thank you in advance

**Comment by ShammyLevva** [comment]
The research suggestions report sort of does this. It’s done specifically for checking census records but at present it shows a colour box to indicate that they are likely to be alive on a date. 

The census tab report does do this more directly showing people you have yet to find on a census who were alive or were likely to be alive on the census date. So in a sense the program already does this just for specific dates census records were taken on 

It’s a reasonable idea and shouldn’t be too hard to achieve as the census report as I say kinda already does this.

**Comment by pcpatfire** [comment]
Thank you for your answer. Yes, the only limitation presently is that you can't go before 1790 (first US Census). It would be great to add a freely selectable date in your Census tab, as an alternative to the fixed census dates. What do your think?

**Comment by ShammyLevva** [comment]
Yes that would be logical to add a date option to that reports pages
